roscala/src/test/scala/coop/rchain/rosette/prim/TupleSpec.scala

```diff
@@ -239,4 +239,56 @@ class TupleSpec extends FlatSpec with Matchers {
     tplTail.fn(newCtxt) should be(Right(Tuple(empty)))
   }
 
+  /** tuple-new */
+  "tplNew" should "correctly create a new Tuple with n Obs" in {
+    val tup = Seq(Fixnum(1), Fixnum(2), Fixnum(3), Fixnum(4), Fixnum(5), Fixnum(6))
+
+    val newCtxt =
+      ctxt.copy(
+        nargs = 8,
```

Comment:
Should be `7`, right?

Reply:
I think so. Done.

Reply:
Nope.  It's because of that ignored first parameter.

Reply:
You still have to count in the ignored first element though.